modules/installation-aws-config-yaml.adoc

```diff
@@ -314,6 +336,16 @@ storage type as `io1` and set `iops` to `2000`.
 ====
 The IMDS configuration for control plane machines that is set during cluster installation can only be changed by using the AWS CLI. The IMDS configuration for compute machines can be changed by using compute machine sets.
 ====
+endif::aws-outposts[]
+ifdef::aws-outposts[]
+<6> For compute instances running in AWS Outpost instance, specify volume type gp2, to avoid using gp3 volume type which is not supported.
+<7> For compute instances running in AWS Outpost instance, specify a supported instance type in the AWS Outpost instance.
+<8> If you provide your own VPC, specify subnets for each availability zone that your cluster uses.
+<9> For compute instances running in AWS Outpost instance, specify the Availability Zone where the Outpost instance located.
```

Comment:
```suggestion
<9> For compute instances running in AWS Outpost instance, specify the Availability Zone where the Outpost is instance located.
```

Reply:
Changed to "... Outpost instance is located"